test/e2e/testing-manifests/storage-csi/hostpath/hostpath/csi-hostpath-testing.yaml

```diff
@@ -0,0 +1,64 @@
+# WARNING: this is only for testing purposes. Do not install in a production
+# cluster.
+#
+# This exposes the hostpath's Unix domain csi.sock as a TCP port to the
+# outside world. The mapping from Unix domain socket to TCP is done
+# by socat.
+#
+# This is useful for testing with csi-sanity or csc.
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: hostpath-service
+spec:
+  type: NodePort
+  selector:
+    app: csi-hostpath-socat
+  ports:
+  - port: 10000 # fixed port inside the pod, dynamically allocated port outside
```

Comment:
Must we deploy this? We've had issues in the past where other e2e tests have chosen conflicting node ports.

Reply:
We don't deploy it (not listed in csi.go). It's here because the script copies the entire directory without any special cases for files that are not needed.